www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -29,106 +29,73 @@ components can communicate with each other, see Application Plugins.
 If you're unfamiliar with Android, you should first familiarize
 yourself with the [Android Platform Guide](index.html) and have the latest Android
 SDK installed before you attempt the more unusual development option
-of embedding a WebView.  Starting with Cordova 1.9, the Android
-platform relies on a `CordovaWebView` component, which builds on a
-legacy `CordovaActivity` component that pre-dates the 1.9 release.
+of embedding a WebView.  Starting with Cordova 4.0, the Android
+platform relies on a `SystemWebView` component.
 
 1. To follow these instructions, make sure you have the latest Cordova
    distribution. Download it from
    [cordova.apache.org](http://cordova.apache.org) and unzip its
```

Comment:
It's hard to find where to download it. Maybe the link should point to dist page https://dist.apache.org/repos/dist/release/cordova/platforms/ or github releases page https://github.com/apache/cordova-android/releases